### PR TITLE
Update introduction page

### PR DESCRIPTION
**Pull request by LachlanCourt**
Introduction page was out of date compared to the Current Members page.
[Preview](https://deploy-preview-147--nubook.netlify.app/team/introduction)

### CODE REVIEW

**Comment by LachlanCourt** [comment]
I accidentally branched off a different branch instead of off master so ignore the commit history lol. It's just this page in this PR

File: src/book/01-team/01-introduction/01-introduction-to-nubots.mdx [hunk 1]
```diff
@@ -18,13 +18,13 @@ The team is led and supported by academics and student team leaders.
...
+- **Josephus Paye II** (Computer Science) - Mentor, Ex-Team Leader
 - **Alex Biddulph** (Computer Engineering) - Mentor, Ex-Team Leader
 - **Dr. Trent Houliston** (Software Engineering) - Mentor, Ex-Team Leader
 
 See [Current Team](/team/current-members) for our current team members and [Past Members](/team/past-members) for past members.
 
 ## The NUbots Lab
 
-The NUbots team works in the Robots and Brain Architecting Lab in ES115A. Feel free to stop by at any time for a tour!
+The NUbots team works in the Robots and Brain Architecture Lab in ES115A. Feel free to stop by at any time for a tour!
```
Comment (CMurtagh-LGTM) [inline src/book/01-team/01-introduction/01-introduction-to-nubots.mdx:30]: Have you read the sign on the lab door?

Comment (CMurtagh-LGTM) [reply src/book/01-team/01-introduction/01-introduction-to-nubots.mdx:30]: From memory, it should something be "Architecturing"

Comment (LachlanCourt) [reply src/book/01-team/01-introduction/01-introduction-to-nubots.mdx:30]: I wasn't sure if it was a typo! I didn't even know Archtitecting was a word 😬

Comment (CMurtagh-LGTM) [reply src/book/01-team/01-introduction/01-introduction-to-nubots.mdx:30]: I'd have to read the sign again, but in these times I cannot.

Comment (LachlanCourt) [reply src/book/01-team/01-introduction/01-introduction-to-nubots.mdx:30]: I'll double check before finalising

Comment (KipHamiltons) [reply src/book/01-team/01-introduction/01-introduction-to-nubots.mdx]: Architecting was correct.

Comment (KipHamiltons) [reply src/book/01-team/01-introduction/01-introduction-to-nubots.mdx]: "Robotics & Brain Architecting Laboratory"

Comment (ysims) [reply src/book/01-team/01-introduction/01-introduction-to-nubots.mdx]: I'd suggest changing it from 'Robot' to 'Robotics', and maybe lab to 'Laboratory' too, in line with what Kip said ^

**Review by CMurtagh-LGTM — APPROVED** [review]
Looks good to me

**Pull request merged**